microbench/db_basic_bench.cc

```diff
@@ -129,6 +246,1019 @@ static void DBClose(benchmark::State& state) {
 BENCHMARK(DBClose)->Iterations(200);  // specify iteration number as the db size
                                       // is impacted by iteration number
 
+static void DBPut(benchmark::State& state) {
+  auto compaction_style = static_cast<CompactionStyle>(state.range(0));
+  uint64_t max_data = state.range(1);
+  uint64_t per_key_size = state.range(2);
+  bool enable_statistics = state.range(3);
+  bool enable_wal = state.range(4);
+  uint64_t key_num = max_data / per_key_size;
+
+  // setup DB
+  static DB* db;
```

Comment:
Is it the intention to make `db` a static variable?

Reply:
Yes, it's used for multi-threaded test, the db is shared between threads: https://github.com/google/benchmark/blob/main/docs/user_guide.md#multithreaded-benchmarks